etl/models/models.go

```diff
@@ -16,15 +16,28 @@ import (
 )
 
 type RecordType string
+type StatesType string
```

Comment:
All this types is really confusing) Maybe here use singular `StateType` and `RequestType` (as analogy to singular `RecordType`), and in const use postfix `Record` (StateRecord, RequestRecord, ResultRecord)?

Reply:
Done